src/components/Stack/Stack.tsx

```diff
@@ -1,20 +1,32 @@
 import React from 'react';
 import classNames from 'classnames';
 import './Stack.scss';
+import * as CSS from 'csstype';
+
+const defaultProperty = 'flex-start';
 
 export type StackProps = React.HTMLAttributes<HTMLDivElement> & {
   space: number;
   direction?: 'vertical' | 'horizontal';
+  alignItems?: CSS.AlignItemsProperty;
+  justifyContent?: CSS.JustifyContentProperty;
+  style?: undefined;
```

Comment:
I wouldn't make this `undefined`, I think we can still allow passing in a `style` attribute even tho we prefer using class names and css instead.

Reply:
This would also make it impossible to use this with https://styled-components.com/ which discovery doesn't do, but why disallow it for all users of `om`?